client/custom_flow.go

```diff
@@ -0,0 +1,103 @@
+package client
+
+type CustomFlowCreatePayload struct {
+	Name                 string           `json:"name"`
+	Repository           string           `json:"repository"`
+	Path                 string           `json:"path,omitempty"`
+	Revision             string           `json:"revision,omitempty"`
+	TokenId              string           `json:"tokenId,omitempty"`
+	SshKeys              []TemplateSshKey `json:"sshKeys,omitempty"`
+	GithubInstallationId int              `json:"githubInstallationId,omitempty"`
+	BitbucketClientKey   string           `json:"bitbucketClientKey,omitempty"`
+	IsBitbucketServer    bool             `json:"isBitbucketServer"`
+	IsGitlabEnterprise   bool             `json:"isGitLabEnterprise"`
+	IsGithubEnterprise   bool             `json:"isGitHubEnterprise"`
+	IsGitLab             bool             `json:"isGitLab"`
+	IsAzureDevOps        bool             `json:"isAzureDevOps" tfschema:"is_azure_devops"`
+	IsTerragruntRunAll   bool             `json:"isTerragruntRunAll"`
+}
+
+type CustomFlow struct {
```

Comment:
Would be great to DRY this a bit and add a shared `struct` between this one and `CustomFlowCreatePayload`

Reply:
They vary from a "tagging" perspective. (json and tfschema).
Trying to merge may become cumbersome, and the benefit is small. It is currently consistent with the rest of the code.